tests/test_tokenization_common.py

```diff
@@ -313,7 +313,7 @@ def convert_batch_encode_plus_format_to_encode_plus(batch_encode_plus_sequences)
     # TODO: this test can be combined with `test_sentencepiece_tokenize_and_convert_tokens_to_string` after the latter is extended to all tokenizers.
     def test_tokenize_special_tokens(self):
         """Test `tokenize` with special tokens."""
-        tokenizers = self.get_tokenizers(fast=True)
+        tokenizers = self.get_tokenizers(fast=True, do_lower_case=True)
```

Comment:
Per #13489, the reason that some special tokens couldn't be correctly tokenized is because they were wrongly lower-cased. So I think enabling `do_lower_case` in the test can cover broader circumstances.

Reply:
I'm really sorry, but I'm not sure I see the tokenizer that would be a problem. Do you remember which one(s) you had the problem with? :slightly_smiling_face:

Reply:
I'm sorry, but I'm not sure I see the tokenizer that would be a problem. Do you remember which one(s) you had the problem with?

Reply:
Please checkout the lines in the link:
https://github.com/qqaatw/transformers/blob/5eeb6e0b9d8a45d036aefcd653a2480e4e75a5b0/src/transformers/tokenization_utils.py#L454-L460

The special tokens added through `tokenizer.add_tokens` were wrongly lower-cased before PR #13489 as they were solely stored in `self.unique_no_split_tokens` variable and not in `self.all_special_tokens`, resulting in lower-cased special tokens which were not intended to be.

Reply:
Got it! Indeed, it makes sens to add this argument! Thanks for clarifying